README.md

```diff
@@ -1756,9 +1756,13 @@ However, this section covers some basics of what to expect with offers, and tips
 
 🔹 Companies can and should work hard to ensure that all candidates are
 [given equal treatment](https://rework.withgoogle.com/guides/pay-equity/steps/introduction/)
-in the hiring process.
-But candidates should also know their worth and understand that they can negotiate to get
-better offers.
+in the hiring process but
+[inequalities persist](https://iwpr.org/publications/gender-wage-gap-2017-race-ethnicity/). [Workplace disparities](https://digitalcommons.ilr.cornell.edu/cgi/viewcontent.cgi?article=2208&context=articles) in pay and opportunity span race and gender, with research focusing on the [US market](http://www.pewresearch.org/fact-tank/2016/07/01/racial-gender-wage-gaps-persist-in-u-s-despite-some-progress/) as a whole,  [executive leadership](https://digitalcommons.ilr.cornell.edu/cgi/viewcontent.cgi?article=2208&context=articles) and its well documented [lack of diversity](http://fortune.com/2017/06/09/white-men-senior-executives-fortune-500-companies-diversity-data/), and the [technology industry](https://www.eeoc.gov/eeoc/statistics/reports/hightech/).
+More effort is needed to close the gap but candidates should know their worth and understand that they can negotiate to get better
```

Comment:
FYI we need to add a linking rule when it comes to using sites (like NYT and NYer) that allow you a certain number of free reads per month before you have to subscribe. Something like no more than four per Guide or something?

Reply:
Added your link. Seems off to say there is gender bias in negotiation but negotiation will get you a better offer. What do you think? (It's in imminent commit)

Reply:
About articles with n-free views til subscription, I agree we need to do something so that we don't have readers eager to read links they can't access, and that could be limiting number of links to specific domains in the guide (though with return use, that won't work for users for very long). I think it would be awesome (though perhaps farfetched) if we make relationships with these sites and offer traffic (perhaps) as a lure for a URL parameter or something that allows users from Holloway to get to the destination url (only).

Reply:
Re: first point, I think (I know!) we can alter the language to give us both. "This is what is expected of you and here are the dangers of doing it exactly right (because "right" means right for white men) and also look pretty but not too pretty" is the synthesis of all advice to women. We have to be very careful with how we word it, but ultimately it's a decision the team makes on each of the links we choose to include: is this information helpful? If the answer is yes, we figure out what complexities it brings up and we embrace them, share them, and clarify to the extent possible. To me, this link is very helpful. What do you think? I can iterate on the language and we can circle back on it together soon.

Reply:
To your second point, and picture me in a pantsuit as I say this, professional working career woman that I am: YAAAAAAAAS. Nothing is farfetched. This is an idea we've batted around before wrt NYT access. Could be farther off, but keep it in the convo. I know @jlevy can tell us some interesting things about the search engineering this would take.

Reply:
I think the article is very helpful and we should work out the language. We can discuss today.